backend/controllers/authController.js

```diff
@@ -225,4 +225,42 @@ const resetPassword = async(req, res) => {
     }
 };
 
-export { register, verifyEmail, login, forgotPassword, verifyResetCode, resetPassword };
\ No newline at end of file
+// Get Profile Handler
+const getProfile = async (req, res) => {
+    try {
+        const user = await UserModel.findById(req.user.id).select('-password'); // Exclude password
```

Comment:
Perfect! it makes no sense to send the user's password to the client, a very common mistake